final/app.js

```diff
@@ -0,0 +1,122 @@
+/*
+GAME RULES:
+
+- The game has 2 players, playing in rounds
+- In each turn, a player rolls a dice as many times as he whishes. Each result get added to his ROUND score
+- BUT, if the player rolls a 1, all his ROUND score gets lost. After that, it's the next player's turn
+- The player can choose to 'Hold', which means that his ROUND score gets added to his GLBAL score. After that, it's the next player's turn
+- The first player to reach 100 points on GLOBAL score wins the game
+
+*/
+
+var scores, roundScore, activePlayer, gamePlaying;
+
+init();
+
+
+document.querySelector('.btn-roll').addEventListener('click', function() {
+    if(gamePlaying) {
+        // 1. Random number
+        var dice = Math.floor(Math.random() * 6) + 1;
+
+        //2. Display the result
+        var diceDOM = document.querySelector('.dice');
```

Comment:
The query selector here is looking for the hard coded value ".dice". As mentioned on the call last week, generally it is best practice to store these variables in a separate file when they can be updated when needed.